### DISCUSSION TITLE
migrare HA Hassio virtualizzato in Qnap verso mini PC

### DISCUSSION THREAD

**Discussion by dibummer** [indomus/forum#1841] [HUB: Home Assistant]
Ciao,
attualmente ho installato HA Hassio virtualizzato su Qnap e vorrei migrare su un mini PC.
non avendo ben chiaro la procedura e  non vorrei perdere tutta la configurazione, qualcuno potrebbe suggerirmi la soluzione?
grazie

**Comment by snakuzzo** [comment]
Dipende da come vuoi utilizzarlo sul minipc.
Se vuoi sempre HA OS, puoi decidere di utilizzarlo come VM su (Virtualbox ad esempio), oppure puoi optare per installazione [HA Supervised](https://indomus.it/guide/installare-e-configurare-home-assistant-supervised-con-docker-su-mini-pc-intel-nuc-computer-con-linux-debian/).

A prescindere da cosa vorrai fare

- fai il backup di HA OS tramite funzionalità apposita su HA
- fai installazione pulita di HA OS (scegli tu se su Virtualbox o HA Supervised)
- quando fai il primo avvio di HA OS ti chiederà se vuoi importare il backup...lo importerai
- Fine

Ovviamente dovrai assicurarti che la configurazione a livello di network sia uguale a quella che hai ora sul QNAP, altrimenti dovrai adeguare qualcosina, in primis a livello di Port forwarding sul router

**Reply by dibummer** [reply]
grazie della fulminea risposta.
vorrei dedicare il mini pc ad home Assistant rimuovendo windows, quindi  ho copiato l'immagine di haos generic 64bit su una chiave usb.
se ho capito bene reinstallo haos sul mini pc e ripristino da un backup. giusto?
grazie

**Reply by snakuzzo** [reply]
Esatto

**Reply by dibummer** [reply]
grazie. gentilissimo.
